src/Symfony/Component/HttpKernel/DependencyInjection/ConfigurableExtension.php

```diff
@@ -12,6 +12,7 @@
 namespace Symfony\Component\HttpKernel\DependencyInjection;
 
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension as DependencyInjectionExtension;
```

Comment:
can't we remove the alias?
```suggestion
use Symfony\Component\DependencyInjection\Extension\Extension;
```

Reply:
I guess you are right